thoth/metrics_exporter/metrics.py

```diff
@@ -98,6 +98,9 @@
 
 graphdb_alembic_table_check = Gauge("thoth_graphdb_alembic_table_check", "alembic version table rows check.", [])
 
+graphdb_last_analysis_datetime = Gauge(
+    "thoth_graphdb_last_analysis_datetime", "Datetime of the last container image analysis synced in the database.", []
```

Comment:
`"thoth_graphdb_last_analysis_datetime", "Datetime of the last container image analysis synced in the database.", ["datetime"]